ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -11,11 +11,14 @@
 
 @{
     var mailtoUrl = "mailto:";
-    
+
     // to: Add the proper support agent as recipient
     mailtoUrl += Model.PuzzleState?.Puzzle.SupportEmailAlias ?? (Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com");
     // cc: Cc the team's email address because what if this email account is a random personal one?
-    mailtoUrl += "?cc=" + Uri.EscapeDataString(Model.Team.PrimaryContactEmail);
+    if (!string.IsNullOrEmpty(Model.Team.PrimaryContactEmail))
+    {
+        mailtoUrl += "?cc=" + Uri.EscapeDataString(Model.Team.PrimaryContactEmail);
+    }
```

Comment:
Oh this was the bug. This needs an else case that adds the `?` there though because everything after this expects a `?` to exist.